src/include/OSL/rendererservices.h

```diff
@@ -29,6 +30,298 @@ typedef const void* TransformationPtr;
 typedef void (*PrepareClosureFunc)(RendererServices*, int id, void* data);
 typedef void (*SetupClosureFunc)(RendererServices*, int id, void* data);
 
+enum class AttributeSpecBuiltinArg {
+    ShaderGlobalsPointer,  // void* (TODO: ideally ShaderGlobals*)
+    ShadeIndex,            // int
+    Derivatives,           // bool
```

Comment:
would
` IsDerivatives,           // bool`
be more self documenting?

Reply:
HasDerivatives?